### ISSUE TITLE
Add `a11y` linting.

### ISSUE THREAD

**Issue by kendrick** [Pier1/rocketbelt#26]
Accessibility is a core tenet of Rocketbelt, but it's not yet baked in. An `a11y` linter needs to be part of a build task so that developing accessible components is easier, and so that a deployment can fail if it contains inaccessible component code.
